stetho/src/main/java/com/facebook/stetho/inspector/elements/Descriptor.java

```diff
@@ -73,13 +73,13 @@ public final void removeCallbacks(Runnable r) {
    * @param text the text argument to be parsed
    * @return a map of attributes to their respective values to be set.
    */
-  protected Map<String, String> parseSetAttributesAsTextArg(String text) {
+  protected static Map<String, String> parseSetAttributesAsTextArg(String text) {
```

Comment:
generally prefer this kind of change in a separate commit, but i'm fine with it for nwo.